Assets/Project/Scripts/GamePlayScene/Bottle/AbstractBottleController.cs

```diff
@@ -141,8 +141,12 @@ public virtual void Initialize(BottleData bottleData)
             BoardManager.Instance.InitializeBottle(this, Id);
 
             // sprite が設定されている場合読み込む
-            if (bottleData.bottleSprite.RuntimeKeyIsValid())
+            if (bottleData.bottleSprite.RuntimeKeyIsValid()) {
                 StartCoroutine(InitializeSprite(bottleData.bottleSprite));
+            } else {
+                GetComponent<Renderer>().sortingLayerName = SortingLayerName.BOTTLE;
+                GetComponent<SpriteRenderer>().enabled = true;
```

Comment:
```suggestion
                var bottleWidth = GetComponent<SpriteRenderer>().sprite.bounds.size.x;
                var bottleHeight = GetComponent<SpriteRenderer>().sprite.bounds.size.y;
                
                transform.localScale = new Vector2(BottleSize.WIDTH / bottleWidth, BottleSize.HEIGHT / bottleHeight);

                if (GetComponent<Collider2D>() is BoxCollider2D) {
                    GetComponent<BoxCollider2D>().size = GetComponent<SpriteRenderer>().sprite.bounds.size;
                }
                GetComponent<Renderer>().sortingLayerName = SortingLayerName.BOTTLE;
                GetComponent<SpriteRenderer>().enabled = true;
```

画像をloadしたときに行っているこの部分も必要じゃない？

Reply:
そうだった。
じゃあ、 `ObjectUnifierにするわ`